src/libbson/src/bson/bson-compat.h

```diff
@@ -171,16 +171,34 @@ typedef signed char bool;
 #endif
 
 
+#if defined(USE_LEGACY_GCC_ATOMICS) || (!defined(__clang__) && \
```

Comment:
Can `defined (USE_LEGACY_GCC_ATOMICS) || ` be removed? I do not see where it is expected to be defined.

Reply:
Done.